aepp/datasets.py

```diff
@@ -88,7 +88,7 @@ def __init__(
             aepp.config.endpoints["global"] + aepp.config.endpoints["dataset"]
         )
 
-    def getLabels(self, dataSetId: str = None) -> dict:
+    def getLabelSchemaTests(self, dataSetId: str = None) -> dict:
```

Comment:
This doesn't seem correct, why are we renaming this function?

Reply:
ya this was an accident, will fix, sorry about that

Reply:
Ok this should be fixed now, can you please recheck this

Reply:
I did not see that change, you probably did not commit to a new PR. 
I am fixing it, I will merge this PR for now, as a start and will fix the typo myself via another commit